extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -256,6 +259,23 @@ export class AmpAnalytics extends AMP.BaseElement {
     return Promise.all(promises);
   }
 
+  handleRequestsFrameUrl_(config) {
+    if (config['requestsFrameUrl'] && config['dataHash']) {
```

Comment:
make sure in the jsdoc that config is explicitly non-nullable, otherwise this predicate can throw